test/cypress/integration/consumer-tools/before-you-claim/before-you-claim.cy.js

```diff
@@ -5,12 +5,14 @@ const claim = new BeforeYouClaim();
 describe( 'Planning your Social Security', () => {
   beforeEach( () => {
     claim.open();
+    claim.interceptRetirementAPIRequests();
```

Comment:
Same as the mapbox api, we should document that the fixture returns a very specific result.

Reply:
I added some comments in 793d321. It's not a very extensive comment since none of our tests actually depend on getting a specific response back, just a valid response (i.e. even if you changed the birthdate and salary the tests would still pass), but hopefully it's enough to point people in the right direction.